Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:5.5-apache
+MAINTAINER Rion Dooley <dooley@tacc.utexas.edu>
+
+# Add project from current repo to enable automated build
+ADD . /var/www/html
+
+# Change ownership for apache happiness
+RUN a2enmod rewrite && chown -R www-data:www-data /var/www/html
```

Comment:
It's a docker container, the underlying OS doesn't matter. This will leverage the trusted php build which, should shrink the overall image size needed to run the stack since both images derive from the same buildroot.